nanoFramework.Graphics/Primitive/Bitmap.cs

```diff
@@ -267,6 +267,17 @@ public void DrawTextInRect(string text, int x, int y, int width, int height, uin
             DrawTextInRect(ref text, ref xRelStart, ref yRelStart, x, y, width, height, dtFlags, color, font);
         }
 
+        /// <summary>
+        /// Draws a single character to the screen
```

Comment:
Missing period and sentence end.

Reply:
Fixed.

Reply:
Thanks! Just missing the native version bump now.